views/submissions.ejs

```diff
@@ -2,10 +2,9 @@
 <html>
 
 <head>
-
 	<title>Submissions | CodeManiacs</title>
 	<meta name="viewport" content="width=device-width, initial-scale=1">
-	<link rel="stylesheet" type="text/css" href="/css/sheet.css">
+	<link rel="stylesheet" type="text/css" href="/css/submissions.css">
 	<link href='https://fonts.googleapis.com/css?family=Sofia' rel='stylesheet'>
```

Comment:
`<link rel="stylesheet" type="text/css" href="/css/sheet.css">` will be there, its for header and footer. And also `<link rel="stylesheet" type="text/css" href="/css/submissions.css">` will be present for the table thing. At least run the app before making the PR.

Reply:
sir app mein user wala chal nhi rha to sirf soch ke pr kr rha hoon

Reply:
aur agar main ejs ko html bnata hoon to usmein header footer nhi aa rhe

Reply:
kyu nhi chal rha `nodemon` run karo... aisa aa raha hai page

![screenshot from 2019-01-28 14-14-13](https://user-images.githubusercontent.com/20883811/51824010-22f90880-2307-11e9-809e-cec52052af89.png)

Reply:
> kyu nhi chal rha `nodemon` run karo... aisa aa raha hai page
> 
> ![screenshot from 2019-01-28 14-14-13](https://user-images.githubusercontent.com/20883811/51824010-22f90880-2307-11e9-809e-cec52052af89.png)

main hi url galat daal rha tha isliye nhi chala

Reply:
issue ke desc mei URL di hui hai https://github.com/KamandPrompt/CodeManiacs/issues/41